tools/reactive-controllers/src/FocusGroup.ts

```diff
@@ -266,6 +265,28 @@ export class FocusGroupController<T extends HTMLElement>
         });
         this.prevIndex = this.currentIndex;
         this.currentIndex = targetIndex > -1 ? targetIndex : this.currentIndex;
+
+        if (this.isRelatedTargetOrContainAnElement(event)) {
+            this.hostContainsFocus();
+        }
+    };
+
+    handleClick = (): void => {
+        // Manually set the tabindex to 0 for the current element on receiving focus (from keyboard or mouse)
+        const elements = this.elements;
+        if (!elements.length) return;
```

Comment:
can we use https://eslint.org/docs/latest/rules/curly? :)

Reply:
we haven't defined `curly` eslint rule in our `.eslintrc.json`. 
1. I can manually rewrite all such statements in the files I touched in PR to follow the rule. (Rest of the repo will still have these statements)
2. I add a rule and make a commit. (Will end up pushing unrelated changes in this PR)
3. Make a separate PR with updated `.eslint.json` file and mark this comment as OOS for this PR

thoughts?

Reply:
Let's not create this rule here in this PR. We can add some additional rules in a follow up PR to align with our linting strategy

Reply:
Yes, no need to create it on this PR, just wanted to write it down for awareness.